docs/guide/getting-started.md

```diff
@@ -26,11 +26,51 @@ if (typeof window.ethereum !== 'undefined') {
 
 You can review the full API for the `window.ethereum` object [here](./ethereum-provider.html).
 
+### Connecting to a test network.
+
+**Show test networks**<br />
+![image](https://user-images.githubusercontent.com/1038445/150907727-77fe4899-5238-4df6-8fa4-e6ae522e7d02.png)<br />
+
+![image](https://user-images.githubusercontent.com/1038445/150907769-c7ea63a8-6bd2-4f80-a376-a70243bb14b7.png)<br />
+
+**Connecting to Ropsten Test Network**<br />
```

Comment:
If we put this here, then it will sound like the only test network that MetaMask officially supports for development purposes is Ropsten. There is nothing wrong with this if this is true — and I think it is — but maybe it's worth adding some words about this after this header in case people are curious? Otherwise I think the instructions would seem very specific for a reason that might not be obvious.